src/chrome/content/rules/Codeforces.ru.xml

```diff
@@ -0,0 +1,27 @@
+
+<!--
+Disabled by https-everywhere-checker because:
```

Comment:
I think this file is unrelated to the PR, would you remove it?

Reply:
Thanks for noticing!